internal/controllers/user_integrations_controller.go

```diff
@@ -1948,6 +1949,14 @@ func (udc *UserDevicesController) registerDeviceTesla(c *fiber.Ctx, logger *zero
 		return err
 	}
 
+	if apiVersion == constants.TeslaAPIV2 {
+		err = udc.registerTeslaForTelemetryServer(c.Context(), region, reqBody.AccessToken, v)
+		if err != nil {
+			// We should consider pushing this to a Queue for retry
+			logger.Err(err).Str("vin", v.VIN).Int("id", v.ID).Msg("Could not register Tesla for Telemetry")
```

Comment:
I don't think we want to log a mapping between people's VIN and their vehicle ID since these logs get stored in elastic with the vehicle data. where we specifically strip out the VIN.